ui/raidboss/data/05-shb/eureka/delubrum_reginae_savage.js

```diff
@@ -33,7 +33,7 @@ const avowedCenterX = -272;
 const avowedCenterY = -82;
 
 // TODO: promote something like this to Conditions?
-const tankBusterOnParty = (data, matches) => {
+const tankBusterOnParty = (data) => {
```

Comment:
Mmmm, I think this is a bug.  Your change is correct, in that matches is not used, but I think it should be used?  It can be fixed up afterwards.